streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -638,33 +578,15 @@ public void shouldThrowIfBuiltInMetricsVersionInvalid() {
         );
     }
 
-    @SuppressWarnings("deprecation")
-    @Test
-    public void shouldResetToDefaultIfConsumerIsolationLevelIsOverriddenIfEosAlphaEnabled() {
-        props.put(StreamsConfig.PROCESSING_GUARANTEE_CONFIG, EXACTLY_ONCE);
-        shouldResetToDefaultIfConsumerIsolationLevelIsOverriddenIfEosEnabled();
-    }
-
-    @SuppressWarnings("deprecation")
-    @Test
-    public void shouldResetToDefaultIfConsumerIsolationLevelIsOverriddenIfEosBetaEnabled() {
-        props.put(StreamsConfig.PROCESSING_GUARANTEE_CONFIG, EXACTLY_ONCE_BETA);
-        shouldResetToDefaultIfConsumerIsolationLevelIsOverriddenIfEosEnabled();
-    }
-
     @Test
     public void shouldResetToDefaultIfConsumerIsolationLevelIsOverriddenIfEosV2Enabled() {
         props.put(StreamsConfig.PROCESSING_GUARANTEE_CONFIG, EXACTLY_ONCE_V2);
-        shouldResetToDefaultIfConsumerIsolationLevelIsOverriddenIfEosEnabled();
-    }
-
-    private void shouldResetToDefaultIfConsumerIsolationLevelIsOverriddenIfEosEnabled() {
         props.put(ConsumerConfig.ISOLATION_LEVEL_CONFIG, "anyValue");
         final StreamsConfig streamsConfig = new StreamsConfig(props);
         final Map<String, Object> consumerConfigs = streamsConfig.getMainConsumerConfigs(groupId, clientId, threadIdx);
         assertThat(
-            consumerConfigs.get(ConsumerConfig.ISOLATION_LEVEL_CONFIG),
-            equalTo(READ_COMMITTED.name().toLowerCase(Locale.ROOT))
+                consumerConfigs.get(ConsumerConfig.ISOLATION_LEVEL_CONFIG),
+                equalTo(READ_COMMITTED.name().toLowerCase(Locale.ROOT))
```

Comment:
nit: 4 spaces are enough for indentation.

Reply:
Done